cmd/options.go

```diff
@@ -91,6 +91,11 @@ func optionFlagSet() *pflag.FlagSet {
 	flags.StringSlice("tag", nil, "add a `tag` to be applied to all samples, as `[name]=[value]`")
 	flags.String("console-output", "", "redirects the console logging to the provided output file")
 	flags.Bool("discard-response-bodies", false, "Read but don't process or save HTTP response bodies")
+	flags.String("dns", lib.DefaultDNSConfig().String(), "DNS configuration. Possible ttl values are: 'inf' "+
+		"for a persistent cache, '0' to disable the cache,\nor a positive duration, e.g. '1s', '1m', etc. "+
+		"Milliseconds are assumed if no unit is provided.\n"+
+		"Possible select values to return a single IP are: 'first', 'random' or 'round-robin'.\n"+
+		"Possible policy values are: 'preferIPv4', 'preferIPv6', 'onlyIPv4', 'onlyIPv6' or 'any'.\n")
```

Comment:
The mixed kebab-case and camelCase for these options is bothering me 😄

Should we rename `policy` to `ip` (short for "IP version preference") and the values to `4`, `6`, `4only`, `6only` and `any`?

Reply:
True, but the mixed casing is still annoying...

Reply:
OK, I made the `select` options camel case as well, so it's `roundRobin` now, but I don't see that enumer feature you mentioned @na--. `-ignorecase` seems to have been added in [a couple of PRs](https://github.com/alvaroloes/enumer/search?q=ignorecase&type=issues), but they weren't merged. Unless you're talking about a different fork, of which there are several.

We might be able to hack around it, but I'd rather leave it as is, even though I prefer lower/kebab case values.

Reply:
https://github.com/dmarkham/enumer/pull/22 and https://github.com/dmarkham/enumer/pull/38 seem to have been the PRs and they are merged

Reply:
but in any case, this can happen in a separate PR, since we use enumer in other places as well